nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/package.json

```diff
@@ -0,0 +1,149 @@
+    "//-04": "The ASF licenses this file to You under the Apache License, Version 2.0",
+    "//-05": "(the \"License\"); you may not use this file except in compliance with",
+    "//-06": "the License.  You may obtain a copy of the License at",
+    "//-07": "",
+    "//-08": "http://www.apache.org/licenses/LICENSE-2.0",
+    "//-09": "",
+    "//-10": "Unless required by applicable law or agreed to in writing, software",
+    "//-11": "distributed under the License is distributed on an \"AS IS\" BASIS,",
+    "//-12": "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.",
+    "//-13": "See the License for the specific language governing permissions and",
+    "//-14": "limitations under the License.",
+    "name": "nifi-registry",
+    "version": "0.0.5-SNAPSHOT",
+    "description": "",
+    "scripts": {
+        "start": "npm run watch",
+        "test": "karma start karma.conf.js --single-run",
+        "test:dev": "karma start karma.conf.js",
+        "eslint": "./node_modules/.bin/eslint './webapp/**/*.js'",
+        "eslint:fix": "./node_modules/.bin/eslint './webapp/**/*.js' --fix",
+        "stylelint": "./node_modules/.bin/stylelint './webapp/theming/**/*.scss'",
+        "lint": "npm run eslint && npm run stylelint",
+        "lint:production": "../../target/frontend-working-directory/node/node ../../target/frontend-working-directory/node/node_modules/npm/bin/npm-cli.js run eslint --scripts-prepend-node-path && ../../target/frontend-working-directory/node/node ../../target/frontend-working-directory/node/node_modules/npm/bin/npm-cli.js run stylelint --scripts-prepend-node-path",
+        "build:webapp:bundle:development": "npm run webpack:development",
+        "build:webapp:bundle:production": "../../target/frontend-working-directory/node/node ../../target/frontend-working-directory/node/node_modules/npm/bin/npm-cli.js run lint:production && ../../target/frontend-working-directory/node/node ../../target/frontend-working-directory/node/node_modules/npm/bin/npm-cli.js run webpack:production",
+        "webpack:development": "./node_modules/.bin/webpack --config webpack.dev.js",
+        "webpack:production": "./node_modules/.bin/webpack --config webpack.prod.js",
+        "watch": "npm run webpack:development && ./node_modules/.bin/webpack-dev-server --config webpack.dev.js",
+        "lint-staged": "./node_modules/.bin/lint-staged",
+        "preinstall": "../../target/frontend-working-directory/node/node ../../target/frontend-working-directory/node/node_modules/npm/bin/npm-cli.js --ignore-scripts install npm-force-resolutions && ../../target/frontend-working-directory/node_modules/.bin/npm-force-resolutions"
+    },
+    "license": "Apache-2.0",
+    "husky": {
+        "hooks": {
+            "pre-commit": "lint-staged"
+        }
+    },
+    "lint-staged": {
+        "*.{js,jsx,ts,tsx}": "eslint",
+        "*.css": "stylelint",
+        "*.scss": "stylelint --syntax=scss"
+    },
```

Comment:
Removing these blocks along with husky from the `dependencies` section should avoid adding the Git hooks.